qa/logging-config/build.gradle

```diff
@@ -5,8 +5,34 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
+apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.legacy-java-rest-test'
 
-apply plugin: 'elasticsearch.standalone-test'
+dependencies {
+  testImplementation project(":libs:elasticsearch-x-content")
+  testImplementation project(":test:framework")
+
+}
+
+testClusters.all {
+  setting 'xpack.security.enabled', 'false'
+}
+
+testClusters.matching { it.name == "javaRestTest" }.configureEach {
+  /**
+   * Provide a custom log4j configuration where layout is an old style pattern and confirm that Elasticsearch
+   * can successfully startup.
+   */
+  extraConfigFile 'log4j2.properties', file('custom-log4j2.properties')
+}
+
+tasks.named("javaRestTest").configure {
+  nonInputProperties.systemProperty 'tests.logfile',
```

Comment:
I think this might make sense to convert the test to the new java rest test framework. Then you won't need these system properties (you can inspect the logs from the test), you don't need to disable security manager (already done for you), and configuration of the cluster sits alongside the test itself. Just a thought.

Reply:
it was tempting to do this, but at the same time I just wanted to 'revert' the commit that removed those tests. I will create an issue to follow up on this